src/app.js

```diff
@@ -0,0 +1,46 @@
+import "regenerator-runtime/runtime";
+import axios from "axios";
+
+const BASE = "https://api.github.com/orgs/stackbuilders/repos";
+
+const getAllItems = async () => {
+  try {
+    const response = await axios.get(BASE);
+
+    const allItems = response.data;
+
+    console.log(moreThanFive(allItems));
+    console.log(lastFiveUp(allItems));
+    console.log(sumStars(allItems));
+  } catch (errors) {
+    console.error(errors);
+  }
+};
+getAllItems();
+// const repos = getAllItems()
+// .then((result) => console.log(result))
+// .catch((err) => {
+// console.error(err);
+// });
+
+const moreThanFive = (repos) => {
+  const fiveStars = repos.filter((repo) => repo.stargazers_count > 5);
+
+  return fiveStars;
+};
+
+const lastFiveUp = (repos) => {
+  const dates = repos
+    .map((repo) => repo.updated_at)
+    .sort()
```

Comment:
🤔 question: Is this sort method working fine?. I might suggest to check if what this is returning is in fact the latest updated. I might be wrong but I think it's worth checking

Reply:
I think that it is right, but why do you think that it is not working?, maybe I am not considering something

Reply:
Yep, seems to be working fine. But I just realized that you're just returning a list of dates, I think you have to return a list of repo objects